emottak-utils/src/main/kotlin/no/nav/emottak/utils/config/KafkaConfig.kt

```diff
@@ -0,0 +1,63 @@
+package no.nav.emottak.utils.config
+
+import java.util.Properties
+
+data class Kafka(
+    val bootstrapServers: String,
+    val securityProtocol: SecurityProtocol,
+    val keystoreType: KeystoreType,
+    val keystoreLocation: KeystoreLocation,
+    val keystorePassword: Masked,
+    val truststoreType: TruststoreType,
+    val truststoreLocation: TruststoreLocation,
+    val truststorePassword: Masked,
+    val groupId: String
+)
+
+fun Kafka.toProperties() = Properties()
+    .apply {
+        put(SECURITY_PROTOCOL_CONFIG, securityProtocol.value)
+        put(SSL_KEYSTORE_TYPE_CONFIG, keystoreType.value)
+        put(SSL_KEYSTORE_LOCATION_CONFIG, keystoreLocation.value)
+        put(SSL_KEYSTORE_PASSWORD_CONFIG, keystorePassword.value)
+        put(SSL_TRUSTSTORE_TYPE_CONFIG, truststoreType.value)
+        put(SSL_TRUSTSTORE_LOCATION_CONFIG, truststoreLocation.value)
+        put(SSL_TRUSTSTORE_PASSWORD_CONFIG, truststorePassword.value)
+    }
+
+data class KafkaEventLoggingProducer(
+    val active: Boolean,
+    val topic: String
+)
+
+@JvmInline
+value class SecurityProtocol(val value: String)
+
+@JvmInline
+value class KeystoreType(val value: String)
+
+@JvmInline
+value class KeystoreLocation(val value: String)
+
+@JvmInline
+value class TruststoreType(val value: String)
+
+@JvmInline
+value class TruststoreLocation(val value: String)
+
+// Kopiert fra hoplite (types.kt), for å forhindre unødvendig stor avhengighet:
```

Comment:
Alle kommentarene i hele denne filen er unødvendige

Reply:
Som utvikler har jeg ikke behov for å vite hvor en kode er kopiert fra. Det samme gjelder dersom jeg delvis kopierer kode fra Stack Overflow eller andre kilder